packages/genesys-spark-components/src/components/beta/gux-prompt-input/gux-prompt-input-beta.tsx

```diff
@@ -82,8 +82,10 @@ export class GuxPromptInputBeta {
     this.isGenerating = false;
   }
 
-  private keyUp(): void {
+  private handleInput(): void {
     this.hasInputText = this.inputElement.value?.length > 0;
+    this.inputElement.parentElement.dataset.replicatedValue =
```

Comment:
Consider adding a comment here about how this is used in the css because it is not obvious outside the context of a PR where you see the css change as well.